articat/meta_artifact.py

```diff
@@ -0,0 +1,25 @@
+from typing import Optional
+
+from articat import Artifact
+from articat.catalog import Catalog
+
+
+class MetaArtifact(Artifact):
+    """Metadata only Artifact. Primary use case is to aggregate information about multiple artifacts."""
+
+    def deps(
+        self, fetch: bool = False, catalog: Optional[type[Catalog]] = None
+    ) -> list[Artifact]:
+        """Returns list of dependencies of this MarkerArtifact"""
+        deps = [
+            Artifact(id=d["id"], partition=d["partition"], version=d["version"])
+            for d in self.metadata.arbitrary["deps"]
+        ]
+        return deps if not fetch else [d.fetch(catalog=catalog) for d in deps]
+
+    def browser_url(self) -> str:
+        """MetaArtifact doesn't really have a proper URL website to open"""
+        return "https://github.com/related-sciences/articat/issues/33"
```

Comment:
So there is not a way to link to the Google Cloud Datastore (if that is the backend) that would show the metadata for the artifact?

Reply:
@dhimmel we could link to datastore, but I'm not sure if it's possible to link to particular value, also would prefer to not link to datastore, I don't think it's very user friendly UI. I will actually change this to be an empty `""` string instead.